docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -61,10 +61,75 @@ I: _I_nfected and currently experiencing a diarrheal disease bout
 
 Data Descriptions
 -----------------
+	 
+	 
+.. list-table:: State definitions and initialization
+   :widths: 5 30
+   :header-rows: 1
+
+   * - State
+     - Definition
+   * - I
+     - Currently has diarrheal disease.
+   * - S
+     - Does not currently have diarrheal disease.
+	 
+.. csv-table:: Prevalence
```

Comment:
Prefer list-table to csv table unless you're loading from data.

Reply:
Why?

Reply:
It's definitely easier to read list-tables when editing; I had been using csvs because I was having issues with formatting the list-tables (which turned out to be due to incorrect spacing)

Reply:
@NathanielBlairStahn Ease of reading in the raw format (which makes them easier to edit) and consistency (which is remarkably important when several people are editing the same source code).  csv-tables if you have actual csv files, list-tables if it fits, full markup tables if you need something fancy.